sklearn/linear_model/coordinate_descent.py

```diff
@@ -17,7 +17,7 @@
 from .base import center_data, sparse_center_data
 from ..utils import check_array, check_X_y, deprecated
 from ..utils.validation import check_random_state
-from ..model_selection import check_cv
+from ..cross_validation import check_cv
```

Comment:
The import path "cross_validation" is deprecated. I am curious to know why you changed to using it.